exercises/day2/exercises/ex_softmax.Rnw

```diff
@@ -1,8 +1,7 @@
 \begin{enumerate}
   \item[a)] What is the relationship between softmax $\pi_k(x)=\frac{\exp(\theta_k^T x)}{\sum^g_{j=1} \exp(\theta_j^T x)}$ and the logistic function $\pix =\frac{1}{1+\exp(\theta^{T} x)}$ for $g = 2$ (binary classification)?
 
-  \item[b)] Derive the loss function of the softmax regression. Suppose there are $n$ instances and $g$ features. (Hint: Apply the same steps as we used in the lecture for the logistic regression)
-How does this loss function relate to the negative log likelihood?
+  \item[b)] Derive the likelihood of $n$ multinomially distributed ($g$ features) target variables when the posterior class probabilities are modeled with softmax regression. How can you transform this likelihood function to an empirical risk function? State the associated loss function. 
```

Comment:
I would give a hint here on the steps. Maybe also a formula they can start with?